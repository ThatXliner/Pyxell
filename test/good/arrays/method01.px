
print [true, true].all(_), [true, false].all(not _)
print [false, true].any(_), [true, true].any(not _)
print [3, -1, 5, 0].filter(_>0)
print [1, 5, -2].map(1+_)
print [0..10].fold(_+_, 0.5)
print [1..10].reduce(_*_), [2].reduce(_//_)
