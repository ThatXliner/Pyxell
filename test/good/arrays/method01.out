true false
true false
[3, 5]
[2, 6, -1]
55.0
3628800 2
